aiida/work/legacy/job_process.py

```diff
@@ -151,3 +151,9 @@ def _setup_db_record(self):
 
         if parent_calc:
             self._calc.add_link_from(parent_calc, "CALL", LinkType.CALL)
+
+        if self.raw_inputs:
```

Comment:
I created a PR now for this part. Is there a reason to have an input be ``None`` for "genuine" purposes, or can we just say that any input which is ``None`` is discarded?